libslide/src/grammar/pattern.rs

```diff
@@ -11,11 +13,12 @@ pub enum ExprPat {
     AnyPat(String),
     BinaryExpr(BinaryExpr<Self>),
     UnaryExpr(UnaryExpr<Self>),
-    Parend(Box<Self>),
-    Braced(Box<Self>),
+    Parend(Rc<Self>),
+    Braced(Rc<Self>),
 }
```

Comment:
I know this is in a previous PR, but why are these called parend and braced instead of paren and bracket.

Reply:
Because they are paranthesized and braced expressions, not a parantheses and bracket.

Reply:
I still don't like this naming. Is this what other libraries use? Braced does not intuitively make me think of a bracket expression.

Reply:
Yeah Braced needs to be changed, can we do it in another pr?

Reply:
yes